src/libs/ajax/ajax-common.ts

```diff
@@ -161,48 +161,6 @@ export const responseContainsRequesterPaysError = (responseText) => {
   return _.includes('requester pays', responseText);
 };
 
-// Allows use of ajaxOverrideStore to stub responses for testing
-const withInstrumentation =
-  (wrappedFetch) =>
-  (...args) => {
-    return _.flow(
-      ..._.map(
-        'fn',
-        _.filter(({ filter }) => {
-          const [url, { method = 'GET' } = {}] = args;
-          return _.isFunction(filter)
-            ? filter(...args)
-            : url.match(filter.url) && (!filter.method || filter.method === method);
-        }, ajaxOverridesStore.get())
-      )
-    )(wrappedFetch)(...args);
-  };
-
-// Ignores cancellation error when request is cancelled
-const withCancellation =
-  (wrappedFetch) =>
-  async (...args) => {
-    try {
-      return await wrappedFetch(...args);
-    } catch (error) {
-      if (error instanceof DOMException && error.name === 'AbortError') {
-        return abandonedPromise();
-      }
-      throw error;
-    }
-  };
-
-// Converts non-200 responses to exceptions
-const withErrorRejection =
-  (wrappedFetch) =>
-  async (...args) => {
-    const res = await wrappedFetch(...args);
-    if (res.ok) {
-      return res;
-    }
-    throw res;
-  };
-
 export const fetchOk = _.flow(withInstrumentation, withCancellation, withErrorRejection)(fetch);
```

Comment:
kept this fetchOk here even though one is available in fetch-core.ts to avoid lingering circular dependency problems that won't be addressed until wave 2 of cleanup.